modules/deleting-cluster.adoc

```diff
@@ -1,22 +1,23 @@
 
 // Module included in the following assemblies:
 //
-// * assemblies/deleting_your_cluster.adoc
+// * osd_quickstart/osd-quickstart.adoc
 
 :_content-type: PROCEDURE
 [id="deleting-cluster_{context}"]
-
 = Deleting your cluster
 
-
 You can delete your {product-title} cluster in the {OCM}.
```

Comment:
Remove 'the' so it's 'OpenShift Cluster Manager'.

Reply:
Thank you for highlighting this. Resolved.